README.md

```diff
@@ -83,7 +83,55 @@ which app id and settings to use. Here is a description of what each means:
 - **fenixBeta** is a release build with beta signing which uses the org.mozilla.fenix.beta app id for beta releases to Google Play
 - **fenixProduction** is a release build with release signing which uses the org.mozilla.fenix app id for production releases to Google Play
 - **fennecProduction** is an experimental build with release signing which uses the org.mozilla.firefox app id and supports upgrading the older Firefox. **WARNING** Pre-production versions of this may result in data loss.
-- **forPerformanceTest** is a release build with the debuggable flag set and test activities enabled for running Raptor performance tests
+- **forPerformanceTest**: see "Performance Build Variants" below.
+
+#### Performance Build Variants
+For accurate performance measurements, read this section!
+
+If you want to analyze performance during **local development**:
+- Recommendation: use a `forPerformanceTest` variant with local Leanplum, Adjust, & Sentry API keys: contact the front-end perf group for access to them
```

Comment:
sorry, but please change `keys` to `tokens`. I know that sounds nitty, but I want to make sure that people realize they are not keys.

Reply:
I can change it but I don't actually know – what is the difference? 😓

Reply:
The difference is only for "semantic" reasons. I am of the opinion that "key" implies the thing is hidden or secret. Because these are not hidden or secure in any way, I don't want to have language that implies otherwise. Does that make sense? Seem too odd? Please let me know!